app/collections/searchCollections.js

```diff
@@ -0,0 +1,29 @@
+import apiClient from 'panoptes-client/lib/api-client';
+
+// Tune this value to determine when a search term is long enough to use Panoptes full-text search.
+const MIN_SEARCH_LENGTH = 3;
```

Comment:
Increasing this value might be a workaround for weird results when the search is 4 or 5 characters.

Reply:
Updating to add that this breaks collections search for Gravity Spy, where there are 2-character collection names.